scripts/deploy-with-s3.js

```diff
@@ -22,16 +22,18 @@ const path = require('path')
 const { execSync } = require('child_process')
 const { remove, move, ensureDir } = require('fs-extra')
 const { s3Prefix, s3Bucket, s3Client } = require('./s3-utils')
-const { cleanUpPageData } = require('./page-data-utils')
 
-const rootDir = path.join(__dirname, '..')
-const cacheDirName = '.cache'
 const publicDirName = 'public'
+const publicDirEntry = [publicDirName, '/']
+const cacheDirEntry = ['.cache', '-cache/']
```

Comment:
This tuple syntax can be changed to something like objects if we're not all into it.